airtable/introduction/table-relationships/airtable-relational-databases.md

```diff
@@ -0,0 +1,38 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Relational Databases
+
+---
+## Content
+
+When thinking of a definition for relational databases, the first thing that comes to mind is that tables in a database contain rows that communicate, hence the relational part. This is not entirely true. The relation is between the column (field), the row, and the data (or type to be more specific) that is inserted. A collection of tables that satisfy the conditions above is called a relational database.
```

Comment:
The last sentence in the paragraph feels a bit redundant, maybe we should remove it? The name of the insight and the first sentence give enough indication that we're talking about relational databases.

> A collection of tables that satisfy the conditions above is called a relational database.